Assets/Scripts/AudioController.cs

```diff
@@ -0,0 +1,87 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.Audio;
+
+public enum MixerGroup
+{
+    Master,
+    SFX,
+    Dialogue,
+    Music
+}
+
+public class AudioController : MonoBehaviour
+{
+    public static AudioController instance;
+
+    [SerializeField]
+    private AudioSource music;
+
+    private AudioSource sound;
+
+    private static AudioMixer mixer;
+    [SerializeField]
+    private AudioMixer inspectorMixer;
+
+    private void Awake()
+    {
+        if (instance == null)
+        {
+            instance = this;
+            mixer = inspectorMixer;
+            DontDestroyOnLoad(gameObject);
+        }
+        else
+        {
+            Destroy(gameObject);
+        }
+    }
+
+    private void Update()
+    {
+        try
+        {
+            sound = GameObject.Find("Sound").GetComponent<AudioSource>();
```

Comment:
just a note that finding game object _every frame_ a second is very taxing. We should find a different way of doing this for the mvp